src/main/java/ch/njol/skript/expressions/ExprCollidableExemptions.java

```diff
@@ -0,0 +1,131 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Name("Collidable Exemptions")
+@Description("Gets/sets/removes entities from the collidable exemptions of a player.")
+@Examples({
+	"send collidable exemptions of player",
+	"add passengers of player and vehicle of player to collidable exemptions of player",
+	"remove target entity from collidable exemptions of player",
+	"reset collidable exemptions of player"
+})
+@Since("INSERT VERSION")
+public class ExprCollidableExemptions extends PropertyExpression<LivingEntity, LivingEntity> {
+
+	static {
+		register(ExprCollidableExemptions.class, LivingEntity.class, "collidable exemptions", "livingentities");
+	}
+
+	@Override
+	protected LivingEntity[] get(Event event, LivingEntity[] source) {
+		List<LivingEntity> entities = new ArrayList<>();
+		for (LivingEntity livingEntity : source){
+			livingEntity.getCollidableExemptions().forEach(uuid -> {
```

Comment:
Can't do a full review rn but this isn't reliable 
This needs to return uuids, because the entities it returns will change based on loaded chunks

Reply:
unresolved